### PR TITLE
170 output to file

### PR DESCRIPTION
**Pull request by dvermd**
This PR adds an option to write output to a file. No coloring are done applied when writing to file.

### CODE REVIEW

**Comment by lu-zero** [comment]
I'd prefer to match hyperfine options so you can have it on screen and on file at the same time.

**Comment by dvermd** [comment]
Just to be clear and avoid another change, you're referencing these options of hyperfine ?
```
    -s, --style <TYPE>
            Set output style type (default: auto). Set this to 'basic' to disable output
            coloring and interactive elements. Set it to 'full' to enable all effects even
            if no interactive terminal was detected. Set this to 'nocolor' to keep the
            interactive output without any colors. Set this to 'color' to keep the colors
            without any interactive output. Set this to 'none' to disable all the output
            of the tool.

        --export-csv <FILE>
        --export-json <FILE>
        --export-markdown <FILE>

        --show-output
```

If that's the case, TYPE should be either `color` or `nocolor` for `av-metrics-tool` ?

**Comment by lu-zero** [comment]
```
        --export-csv <FILE>
        --export-json <FILE>
        --export-markdown <FILE>
```

`--style` can be yet another PR :)

**Review by lu-zero — APPROVED** [review]
Looks good to me, thank you!

File: av_metrics_tool/src/main.rs [hunk 1]
```diff
@@ -41,36 +39,55 @@ fn main() {
...
+        )
+        .arg(
+            Arg::with_name("FILE")
+                .help("Output results to a file")
+                .long("export-file")
+                .takes_value(true)
+                .value_name("FILE"),
+        )
+        .arg(
+            Arg::with_name("OUTPUT")
```
Comment (lu-zero) [inline av_metrics_tool/src/main.rs:54]: I would make it it a --quiet option instead, but it can be done later.

**Pull request merged**